prv_accountant/privacy_random_variables/abstract_privacy_random_variable.py

```diff
@@ -49,7 +49,11 @@ def __init__(self, prv, t_min: float, t_max: float) -> None:
         self.remaining_mass = self.prv.cdf(t_max) - self.prv.cdf(t_min)
 
     def mean(self) -> float:
-        points = [self.t_min, -1e-1, -1e-2, -1e-3, -1e-4, -1e-5, 1e-5, 1e-4, 1e-3, 1e-2, 1e-1, self.t_max]
+        # determine points based on t_min and t_max
+        lower_exponent = int(np.log10(np.abs(self.t_min)))
+        upper_exponent = int(np.log10(self.t_max))
+        points = np.concatenate([[self.t_min], -np.logspace(start=lower_exponent, stop=-5, num=10), [0],
+                                 np.logspace(start=upper_exponent, stop=-5, num=10)[::-1], [self.t_max]])
```

Comment:
Swapping `start` and `stop` should be equivalent to `[::-1]` and is a bit simpler. Is there an edge case where they are not equivalent?

Reply:
Yes, you are right. Thanks!

I fixed it with another commit.